templates/web/base/report/new/_category_select.html

```diff
@@ -1,5 +1,8 @@
 [%~ FOREACH group IN category_groups ~%]
-    [% IF group.name %]<optgroup label="[% group.name %]">[% END %]
+    [% IF group.name %]<optgroup label="[% group.name %]">
+        [% SET pseudo_category = "group-" _ group.name %]
+        <option value="group-[% group.name %]" [% ' selected' IF display_categories.$pseudo_category %]>All [% group.name %]</option>
```

Comment:
This template is included by the admin report edit category dropdown, and the old-app category select, but you only want this change to apply to the dashboard usage - pass in an `include_group_option` in the `INCLUDE` and use that?